### PR TITLE
Tundari Patch

### PR DESCRIPTION
**Pull request by rosebrallier**
Updated Tundari Species for clarity, spelling and grammar as well as:
-Wrote traits into more mechanical terms
-Expanded Tribe descriptions of the Sang'mau and Chi'mau
-Expanded 'krou title for clarity

### CODE REVIEW

File: CharacterCreation/01_03_Species_Tundari.txt [hunk 1]
```diff
@@ -11,19 +11,19 @@ grace and ferocity.
 	The Tundari are a species of obligate carnivores from the planet Tundaar. 
 Physically, they span a wide size range, from the Chi’mau, who can range from 
 3’6” to 5’ and 85-130 lbs to the Sang’mau, at 6’ to 7’ and up to 400 lbs, and 
-every size in-between. Like humans, body types persist in many forms.
+every size in-between. Like humans, Tundari have many different body types.
```
Comment (bleehu) [inline CharacterCreation/01_03_Species_Tundari.txt:14]: much easier to read!

File: CharacterCreation/01_03_Species_Tundari.txt [hunk 2]
```diff
@@ -11,19 +11,19 @@ grace and ferocity.
...
 
-	As a carnivorous species, they have large, forward, wide-set eyes, clawed 
-limbs and large, sharp teeth. Their ears sprout from closer to the top of their 
+	As a carnivorous species, they have large, forward-facing, wide-set eyes, clawed 
+limbs, and large, sharp teeth. Their ears sprout from closer to the top of their 
 heads than the sides, and can be triangular or round. 
 	
 	They are considered a hardy species, but have not adapted well to fight off 
-poisons or foreign contaminants. They are diurnal, and have low-light eyesight, 
+poisons or foreign contaminants. They are diurnal, and have low-light vision,
```
Comment (bleehu) [inline CharacterCreation/01_03_Species_Tundari.txt:26]: much better grammar!

File: CharacterCreation/01_03_Species_Tundari.txt [hunk 3]
```diff
@@ -154,22 +157,23 @@ leaner than the Sang’mau, as a whole.
...
-[Round movement speed calculations up]
+[To calculate your Movement Speed per action, use (Movement Speed)/2 and round up to the
+nearest integer.]
 
 
 Wide-faced:
 	Many species seem to be drawn to your large eyes. You aren’t certain why, but it gives 
 you an advantage when attempting to persuade or charm members of other biological species. 
-[Persuade at Advantage once per biological individual]
+[Once per biological individual, you may roll a Persuation check with Advantage.]
```
Comment (bleehu) [inline CharacterCreation/01_03_Species_Tundari.txt:176]: Much better prose!

File: CharacterCreation/01_03_Species_Tundari.txt [hunk 4]
```diff
@@ -154,22 +157,23 @@ leaner than the Sang’mau, as a whole.
...
+figure-head the process to outsiders.
 
 == Available Traits ==
 
 Hind-Claws:
 	Digging into the ground with the large claws present on your hind-limbs spurs your 
 ability to run much faster than your peers. 
-[Round movement speed calculations up]
+[To calculate your Movement Speed per action, use (Movement Speed)/2 and round up to the
+nearest integer.]
```
Comment (bleehu) [inline CharacterCreation/01_03_Species_Tundari.txt:170]: Much clearer!

File: CharacterCreation/01_03_Species_Tundari.txt [hunk 5]
```diff
@@ -119,25 +119,28 @@ subsequent rolls regarding complications or recovery.
...
 	Note: Tails cannot bear the hanging weight of a Large individual.
 
 Toothed: 
 	As a predatory species, you have the customary long teeth that frighten many. 
-To you, they are commonplace, but to many others, they are far from ordinary. 
-[Gain Advantage on initial Intimidation Rolls toward and against Biological beings]
+To you, they are commonplace, but to many others, they induce a primal fear of being
+hunted and eaten. 
+[Gain Advantage on initial Intimidation checks toward and against non-Tundari Biological
+beings.]
```
Comment (bleehu) [inline CharacterCreation/01_03_Species_Tundari.txt:143]: NB: Why wouldn't being toothy work to intimidate other Tundari? Humans are more intimidated by other big, ugly humans.

**Review by bleehu — APPROVED** [review]
Shipit! :shipit:

**Review by Reveraine — APPROVED** [review]
Yeees!
(Thanks for fixing my grammar!)

File: CharacterCreation/01_03_Species_Tundari.txt (hunk 5 above)
Comment (rosebrallier) [reply CharacterCreation/01_03_Species_Tundari.txt:143]: Fair point.

**Pull request merged**